### DISCUSSION TITLE
I'm sitting in my school but I can't sign up for github education :( What can I do now ?

### DISCUSSION THREAD

**Discussion by minhdaolesoez** [community/community#128237] [GitHub Education]
### Hi everyone,

I'm sitting in my school but I can't sign up for github education :( What can I do now ?

**Comment by samus-aran** [comment]
Hi @minhdaolesoez  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
